src/components/EntryForm.js

```diff
@@ -23,114 +23,193 @@ export default function EntryForm({ onCreateCard }) {
-        <Fieldset>
-          <Part>
-            <label htmlFor="date">Date</label>
-            <Input
-              id="date"
-              name="date"
-              type="date"
-              max={moment().format('YYYY-MM-DD')}
-              onChange={handleOnChange}
-              required
-            />
-          </Part>
-          <Part>
-            <label htmlFor="time">Time</label>
-            <Input
-              id="time"
-              name="time"
-              type="time"
-              onChange={handleOnChange}
-              required
-            />
-          </Part>
-          <Part>
-            <label htmlFor="target">Target species</label>
-            <Input
-              id="target"
-              name="target"
-              type="text"
-              maxLength={25}
-              onChange={handleOnChange}
-              required
-            />
-          </Part>
-        </Fieldset>
-        <small>water</small>
-        <Fieldset>
-          <Part>
-            <label htmlFor="water">water</label>
-            <Input
-              id="water"
-              name="water"
-              type="text"
-              maxLength={25}
-              onChange={handleOnChange}
-              placeholder="water"
-              required
-            />
-          </Part>
-          <Part>
-            <label htmlFor="stretch">Stretch</label>
-            <Input
-              id="stretch"
-              name="stretch"
-              type="text"
-              maxLength={100}
-              onChange={handleOnChange}
-            />
-          </Part>
-          <Part>
-            <label htmlFor="watertemp">Water temperature</label>
-            <Input
-              id="watertemp"
-              name="watertemp"
-              type="number"
-              min={-5}
-              max={50}
-              onChange={handleOnChange}
-              placeholder="°C"
-            />
-          </Part>
-          <Part>
-            <label htmlFor="watercolor">Water Color</label>
-            <select
-              id="watercolor"
-              name="watercolor"
-              type="text"
-              onChange={handleOnChange}
-            >
-              <option value="cloudy">cloudy</option>
-              <option value="normal">normal</option>
-              <option value="clear">clear</option>
-            </select>
-          </Part>
-          <Part>
-            <label htmlFor="waterlevel">Water Level</label>
-            <select
-              id="waterlevel"
-              name="waterlevel"
-              type="text"
-              onChange={handleOnChange}
-            >
-              <option value="low">low</option>
-              <option value="normal">normal</option>
-              <option value="high">high</option>
-            </select>
-          </Part>
-        </Fieldset>
-        <SubmitButton text="Submit" isAccent={true} />
-      </Form>
-    </div>
+    <Form
+      aria-labelledby="formHeader"
```

Comment:
I can't find an element with id 'formHeader' where the aria-label could get its information. Maybe you could link one of your form subjects or add a title to the form? Or maybe I'm not gettin it 😄